src/excalidraw-app/collab/CollabWrapper.tsx

```diff
@@ -76,12 +69,14 @@ class CollabWrapper extends PureComponent<Props, State> {
   private unmounted: boolean;
   private excalidrawRef: any;
   excalidrawAppState?: AppState;
-  private initialData: ImportedDataState;
-  private isCollabScene: boolean;
   private lastBroadcastedOrReceivedSceneVersion: number = -1;
+  private onChangeEmitter = new Emitter<
+    [readonly ExcalidrawElement[], AppState]
+  >();
```

Comment:
we'll want to factor this out to `ExcalidrawApp` when we migrate the collab logic into a hook or something more easily integrate-able with the main app.

Reply:
Yes but that helps when we are trying to listen to changes which will trigger rerendering of the App(for example due to prop drilling) or there are lots of places (which I think for our use case would be 2, one in excal-app and one in collab), in this case, the callback can be triggered from excalidraw-app for collab as well so there won't be any unnecessary rerendering as such.